docs/api/session.md

```diff
@@ -660,8 +660,8 @@ The `proxyBypassRules` is a comma separated list of rules described below:
    Match all hostnames that match the pattern HOSTNAME_PATTERN.
 
    Examples:
-     "foobar.com", "*foobar.com", "*.foobar.com", "*foobar.com:99",
-     "https://x.*.y.com:99"
+     "foobar.com", "\*foobar.com", "\*.foobar.com", "\*foobar.com:99",
+     "https://x.\*.y.com:99"
```

Comment:
This will fix a rendering issue on the docs website since these asterisks were being interpreted as meaning emphasis.

<img width="603" alt="Screenshot 2023-03-31 at 3 10 21 PM" src="https://user-images.githubusercontent.com/5820654/229241795-1cf64d88-69af-46cb-8805-61fdc27ceca6.png">